packages/mattermost-redux/src/action_types/worktemplates.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
In a similar vein, these file and folder names should spell work templates as `work_templates`

Reply:
+1 on this